debug/gdbdiff.ts

```diff
@@ -50,10 +49,15 @@ function printFlags(xpsr: number) {
   return `[${negative}${zero}${carry}${overflow}]`;
 }
 
-function compareRegisters(emulator: Uint32Array, silicone: Uint32Array) {
+async function compareFixRegisters(
+  tofixclient: GDBClient,
```

Comment:
please follow the variable name convention, e.g. `toFixClient` (camel-case).
Also, it makes more sense to me to put this argument last (because the method name if compareFix, I expect the first arguments to be related to compare, and then the next ones to be compared to the fix). I Hope this makes sense